pyphare/pyphare/pharesee/run/utils.py

```
@@ -59,43 +65,41 @@ def _current2d(bx, by, bz, dx, dy):
     return jx, jy, jz
 
 
-def _compute_current(patchdatas, **kwargs):
-    reference_pd = patchdatas["Bx"]  # take Bx as a reference, but could be any other
+def _compute_current(patch, **kwargs):
```

Ruff (0.14.13)

Unused function argument: `kwargs`

(ARG001)